keyring_windows.go

```diff
@@ -1,10 +1,25 @@
 package keyring
 
 import (
-	"github.com/danieljoos/wincred"
 	"syscall"
+
+	"github.com/danieljoos/wincred"
 )
 
+// ignore, just for typecast in keyring.go
+type secretServiceProvider struct {
+	keyringName string
+}
+
+// Set password in keyring for user.
+func (s secretServiceProvider) Set(service, user, password string) error { return nil }
+
+// Get password from keyring given service and user name.
+func (s secretServiceProvider) Get(service, user string) (string, error) { return "", nil }
```

Comment:
Should not this return DefaultKeyringName?

Reply:
Those functions aren't getting called at all. Those are just for the compiler to not throw an error. Otherwise on any other GOOS than linux the compiler would say that there is no such secretServiceProvider to typecast to. 
Same in `keyring_darwin.go`

Reply:
The DefaultKeyring is picked here:
https://github.com/JojiiOfficial/go-keyring/blob/master/secret_service/secret_service.go#L110

Reply:
For me it looks like you should refactor this part, such that the empty string check should be dropped and the default should be returned by the new functions created.

@mikkeloscar Wdyt?